job-dsls/jobs/new_compile_downstream_build.groovy

```diff
@@ -0,0 +1,171 @@
+/**
+ * Creates compile downstream pullrequest (PR) jobs for kiegroup GitHub org. units.
+ * These jobs execute the downstream build skipping tests and skipping the gwt compilation
+ * for a specific PR to make sure the changes do not break the downstream repos.
+ */
+
+import org.kie.jenkins.jobdsl.Constants
+
+def final DEFAULTS = [
+        ghOrgUnit              : Constants.GITHUB_ORG_UNIT,
+        branch                 : Constants.BRANCH,
+        timeoutMins            : 650,
+        label                  : "kie-rhel7 && kie-mem16g",
+        ghAuthTokenId          : "kie-ci-token",
+        ghJenkinsfilePwd       : "kie-ci",
+        artifactsToArchive     : [
+                ",**/target/screenshots/**",
+                "**/target/kie-server-*ee7.war",
+                "**/target/kie-server-*webc.war",
+                "**/target/jbpm-server*dist*.zip",
+                "**/target/business-monitoring-webapp.war",
+                "**/target/business-central*wildfly*.war"
+        ]
+]
+// override default config for specific repos (if needed)
+def final REPO_CONFIGS = [
+        "lienzo-core"               : [],
+        "lienzo-tests"              : [],
+        "droolsjbpm-build-bootstrap": [],
+        "kie-soup"                  : [],
+        "appformer"                 : [],
+        "droolsjbpm-knowledge"      : [],
+        "drools"                    : [],
+        "optaplanner"               : [],
+        "jbpm"                      : [],
+        "kie-jpmml-integration"     : [],
```

Comment:
i think this repository is also leaf and no other repo depends on it. 
Is that correct @danielezonca ?

Reply:
any feedback about kie-jpmml-integration @danielezonca ?

Reply:
@mareknovotny 
Sorry I missed the previous message.
You are right, no other repo depends on it (actually this must not change because of AGPL license AGPL)

Reply:
@danielezonca my question was more about to not include CDB for kie-jpmml-integration changes

Reply:
@mareknovotny 
I don't know if I get completely the question but this repo should be part of FDC because a change in DMN api can break It.
Btw it is a leaf project so it could make sense to invoke a FDC from an upstream project while of course it makes no sense to invoke a FDC from this repo